piff/psf.py

```diff
@@ -116,4 +116,29 @@ def read(cls, file_name, logger=None):
                 logger.debug("interp = %s",interp)
         return cls(model,interp)
 
+    def drawImage(self, position, offset=None, logger=None, **kwargs):
+        """Generates star given position and galsim Image
 
+        :param position:    Position we are interpolating to.
+        :param offset:      Tuple; sett the nominal center of the image
+                            relative to actual center of image [Default: None]
+        :param logger:      A logger object for logging debug info. [default: None]
+
+        :returns: a galsim image
+        """
+        import galsim
+        image = galsim.Image(32, 32)
```

Comment:
Completely arbitrary! I think for our purposes it doesn't matter _that much_, and we could always make it a part of the kwargs for drawImage instead